src/server/scripts/ExilesReach/zone_exiles_reach.cpp

```diff
@@ -4114,6 +4296,616 @@ class spell_tutorial_health_dnt : public SpellScript
+            Creature* garrickPersonal = garrick->SummonPersonalClone(garrick->GetPosition(), TEMPSUMMON_MANUAL_DESPAWN, 0s, 0, 0, player);
+            if (!huxsworthPersonal || !garrickPersonal)
+                return;
+
+            if (npc_huxsworth_briarpatch_quest_private* personalAI = CAST_AI(npc_huxsworth_briarpatch_quest_private, huxsworthPersonal->AI()))
+            {
+                personalAI->SetGarrickGUID(garrickPersonal->GetGUID());
+            }
+        }
+    }
+
+    void Register() override
+    {
+        OnEffectHit += SpellEffectFn(spell_validated_quest_accept_briarpatch_alliance::HandleScript, EFFECT_0, SPELL_EFFECT_SEND_EVENT);
+    }
+};
+
+// 325309 - Validated Quest Accept
+class spell_validated_quest_accept_briarpatch_horde : public SpellScript
+{
+    void HandleScript(SpellEffIndex /*effIndex*/)
+    {
+        if (Player* player = GetCaster()->ToPlayer())
+        {
+            Creature* dawntracker = FindCreatureIgnorePhase(player, "dawntracker_briarpatch", 10.0f);
+            Creature* grimaxe = FindCreatureIgnorePhase(player, "grimaxe_briarpatch", 10.0f);
+            if (!dawntracker || !grimaxe)
+                return;
+
+            Creature* dawntrackerPersonal = dawntracker->SummonPersonalClone(dawntracker->GetPosition(), TEMPSUMMON_MANUAL_DESPAWN, 0s, 0, 0, player);
+            Creature* grimaxePersonal = grimaxe->SummonPersonalClone(grimaxe->GetPosition(), TEMPSUMMON_MANUAL_DESPAWN, 0s, 0, 0, player);
+            if (!dawntrackerPersonal || !grimaxePersonal)
+                return;
+
+            if (npc_dawntracker_briarpatch_quest_private* personalAI = CAST_AI(npc_dawntracker_briarpatch_quest_private, dawntrackerPersonal->AI()))
+            {
+                personalAI->SetGrimaxeGUID(grimaxePersonal->GetGUID());
+            }
+        }
+    }
+
+    void Register() override
+    {
+        OnEffectHit += SpellEffectFn(spell_validated_quest_accept_briarpatch_horde::HandleScript, EFFECT_0, SPELL_EFFECT_SEND_EVENT);
+    }
+};
+
+enum Geolord
+{
+    CONVERSATION_GEOLORD_AGGRO = 13712,
+
+    CONVERSATION_ACTOR_GEOLORD = 70670,
+    CONVERSATION_ACTOR_LINDIE  = 71238,
+    CONVERSATION_ACTOR_CORK    = 75976,
+
+    EVENT_CAST_EARTH_BOLT      = 1,
+    EVENT_CAST_UPHEAVAL        = 2,
+
+    NPC_CORK_FIZZLEPOP         = 167008,
+    NPC_LINDIE_SPRINGSTOCK     = 154301,
+
+    SET_DATA_FREE_PRISONER     = 1,
+
+    SPELL_NECROTIC_RITUAL_DNT  = 305513,
+    SPELL_EARTH_BOLT           = 270453,
+    SPELL_UPHEAVAL             = 319273
+};
+
+// This script is used by Geolord Grek'og for Down with the Quilboar quest
+struct npc_geolord_grekog : public ScriptedAI
+{
+    npc_geolord_grekog(Creature* creature) : ScriptedAI(creature) { }
+
+    void JustAppeared() override
+    {
+        uint32 _entry = NPC_LINDIE_SPRINGSTOCK;
+        std::string_view creatureString = "briarpatch_bunny_alliance";
+
+        if (me->GetInstanceId())
```

Comment:
i dont think this is the correct check we want here; check for worldstate to know if its alliance/horde map; just like we are doing it for the spawngroups in conditions

Reply:
Don't know how to do it.

Reply:
fixed